docs/internals/deprecation.txt

```diff
@@ -103,6 +103,9 @@ details on these changes.
 See the :ref:`Django 3.1 release notes <deprecated-features-3.1>` for more
 details on these changes.
 
+* The ability to use boolean value in :attr:`BaseCommand.requires_system_checks
+  <django.core.management.BaseCommand.requires_system_checks>` will be removed.
```

Comment:
This should be moved to features removed in Django 4.1.

Reply:
I moved it to features removed in Django 4.1 and also replaces all `RemovedInDjango40Warning` with `RemovedInDjango41Warning`